Author: Alastair McKinstry <mckinstry@debian.org>
Description: For some reason, top_srcdir support is broken in auto*. Disable tests.
Last-Updated: 2013-11-15
Forwarded: no

Index: cdo-1.6.2+dfsg.1/test/Makefile.am
===================================================================
--- cdo-1.6.2+dfsg.1.orig/test/Makefile.am	2013-11-15 12:01:13.000000000 +0000
+++ cdo-1.6.2+dfsg.1/test/Makefile.am	2013-11-15 12:02:01.000000000 +0000
@@ -2,11 +2,14 @@
 
 export
 
-TESTS = $(top_srcdir)/test/test_Gridarea.sh \
-        $(top_srcdir)/test/test_Remap.sh \
-        $(top_srcdir)/test/test_Spectral.sh \
-        $(top_srcdir)/test/test_Timstat.sh \
-        $(top_srcdir)/test/test_Vertint.sh
+# Disable any reference to top_srcdir in tests
+# or configure will break.
+# -- amck, 2013-11-15
+#TESTS = $(top_srcdir)/test/test_Gridarea.sh \
+#        $(top_srcdir)/test/test_Remap.sh \
+#        $(top_srcdir)/test/test_Spectral.sh \
+#        $(top_srcdir)/test/test_Timstat.sh \
+#        $(top_srcdir)/test/test_Vertint.sh
 
 #       $(top_srcdir)/test/test_info.py
 #	$(top_srcdir)/test/test_diff.py
